packages/react-components/react-tooltip/package.json

```diff
@@ -38,6 +38,7 @@
     "@fluentui/react-shared-contexts": "^9.3.3",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

Comment:
_nit:_ Can you put this dependency above `@fluentui/react-portal` in line `36` to keep alphabetical order?